frontend/typescript/index.js

```diff
@@ -15,6 +15,8 @@ module.exports = {
             files: ["**/*.tsx"],
             rules: {
                 "complexity": ["error", 10],
+                "react/prop-types": "off",
```

Comment:
**N**: Personally, I think we should keep this on for html elements to avoid unnecessary console log warnings